main.go

```diff
@@ -96,6 +96,13 @@ func main() {
 					return getBalance(ctx.Context, bs, tsk, actorAddr)
 				},
 			},
+			{
+				Name:        "fil-to-eth-address",
+				Description: "Converts an fX address to a 0x one if possible. Either pass a state CAR, tipset CIDs, or trust chain.love for a recent time",
```

Comment:
Wondering if I should add caveats here about waiting "long enough"? Seems like if you're going to trust an RPC endpoint then asking about 2hrs ago should be fine (given conversations with @ribasushi and results from https://filecoinproject.slack.com/archives/C02D73MHM63/p1718693790469889) https://github.com/aschmahmann/filexp/blob/f52cfc729ffdb51d6b97709d8f3fc911dd38820d/main.go#L188

Maybe more warning should be given if you're passing tipset CIDs or a state CAR that you should chose an "old enough one". While this is true for other commands (e.g. get-balance) the footguns/confusion here might be larger, not sure.

WDYT @ribasushi?